PayPal/src/main/java/com/braintreepayments/api/PayPalRequest.java

```diff
@@ -239,75 +163,27 @@ public PayPalRequest intent(@PayPalPaymentIntent String intent) {
      *
      * @see <a href="https://developer.paypal.com/docs/api/payments/v1/#definition-application_context">See "landing_page" under the "application_context" definition</a>
      */
-    public PayPalRequest landingPageType(@PayPalLandingPageType String landingPageType) {
+    public void setLandingPageType(@PayPalLandingPageType String landingPageType) {
         mLandingPageType = landingPageType;
-        return this;
-    }
-
-    /**
-     * Set the checkout user action which determines the button text.
-     *
-     * @param userAction Must be a be {@link PayPalPaymentUserAction} value:
-     * <ul>
-     * <li>{@link #USER_ACTION_COMMIT}</li>
-     * <li>{@link #USER_ACTION_DEFAULT}</li>
-     * </ul>
-     *
-     * @see <a href="https://developer.paypal.com/docs/api/payments/v1/#definition-application_context">See "user_action" under the "application_context" definition</a>
-     */
-    public PayPalRequest userAction(@PayPalPaymentUserAction String userAction) {
-        mUserAction = userAction;
-        return this;
-    }
-
-    /**
-     * Offers PayPal Credit prominently in the payment flow. Defaults to false. Only available with Billing Agreements
-     * and PayPal Checkout.
-     *
-     * @param offerCredit Whether to offer PayPal Credit.
-     */
-    public PayPalRequest offerCredit(boolean offerCredit) {
```

Comment:
On iOS - we ended up adding `offerCredit` back in for an LE merchant. I assume we'll want to keep it around for Android too?